setup.py

```diff
@@ -12,16 +12,14 @@
     history = history_file.read()
 
 install_requires = [
-    "matplotlib>=3.2.0,<3.4;python_version<'3.7'",
     "matplotlib>=3.4.0,<4;python_version>='3.7' and python_version<'3.10'",
     "matplotlib>=3.6.0,<4;python_version>='3.10'",
-    "numpy>=1.18.0,<1.20.0;python_version<'3.7'",
     "numpy>=1.20.0,<2;python_version>='3.7' and python_version<'3.10'",
```

Comment:
should we drop support for numpy<1.22 because of these security warnings? https://github.com/sdv-dev/Copulas/security/dependabot

Reply:
Last supported version for numpy and python 3.7 is `1.21.6` 😥